test/gurka/test_geometry.cc

```diff
@@ -0,0 +1,153 @@
+#include "gurka.h"
```

Comment:
this test doesnt use gurka at all, so i dont think it needs to be a gurka test. in fact maybe it should be added to an existing test?

Reply:
Correct.  I can move it.